hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestPipelineStateManagerImpl.java

```diff
@@ -295,37 +295,36 @@ public void testGetPipelinesByTypeFactorAndState()
 
   @Test
   public void testAddAndGetContainer() throws IOException, TimeoutException {
-    long containerID = 0;
```

Comment:
May I ask what is the rationale behind changing this long to AtomicLong? The rest of the patch is really just changing exception testing to assertThrows, is it by chance a leftover from some other work, or intentional?

Reply:
I assume it is changed to be able to use in this lambda expression:

https://github.com/apache/ozone/blob/482ec1ac126e9b27b8d4659be56f09785eed28a3/hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestPipelineStateManagerImpl.java#L322-L325

I think it's fine.  If we want to minimize the diff, we could keep `long containerID` and change the code above to something like this:

```java
    ContainerID cid = ContainerID.valueOf(++containerID);
    IOException e = assertThrows(IOException.class,
        () -> stateManager.addContainerToPipeline(finalPipeline.getId(), cid));
```

Reply:
@adoroszlai That's a good idea. I will modify it. Thanks

Reply:
thank you for the explanation, I did not catch that point. my bad.